### PR TITLE
feat(scripts): add scripts for updating kms key policy

### PR DESCRIPTION
**Pull request by kajm**
**What was done**
Added bash script for updating the KMS key policy for AWS EventBridge, which in turns enables SQS SSE-KMS encryption.

**How was it done**
Added a script which puts a new key policy for the external/master customer provided KMS key. 
Also updated the README file

### CODE REVIEW

File: scripts/put-kms-key-policy.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+if [ -z ${KEY_ID} ]; then
+    echo "No key id found for key with alias ${KEY_ALIAS}"
+    exit 1
+fi
+
+aws kms put-key-policy \
+    --policy-name "${KEY_POLICY_NAME}" \
+    --key-id "${KEY_ID}" \
+    --policy "${POLICY}"
```
Comment (MasterKenth) [inline scripts/put-kms-key-policy.sh:76]: Could add an `echo success` after, optionally checking the exit code `$?` of `aws kms ...` to see if it actually succeeded.

File: README.md [hunk 2]
```diff
@@ -200,6 +201,19 @@ Following this example, the arn would be
 arn:aws:kms:${AWS::Region}:${AWS::AccountId}:alias/external/master
 ```
 
+### Allow AWS EventBridge to send messages to encrypted SQS
+In order to let AWS EventBridge put messages (events) on AWS SQS that has server-side-encryption enabled, 
+the KMS key policy for external/master key needs to be updated. 
+
+Go to the `scripts` catalog and simply run the `put-kms-key-policy` script
```
Comment (MasterKenth) [inline README.md:208]: Mega-minor nitpick, add ":" after "script".

File: scripts/put-kms-key-policy.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+clear
+echo This script puts a new key policy on the customer provided AWS KMS key
+echo named external/master. The policy allows AWS EventBridge to put messages
+echo on AWS SQS when encryption is enabled with the customer provided KMS key.
+echo
+echo The currently active source account is \"${AWS_VAULT:Unknown}\"
+echo
+read -p "Type Y to continue: " CHOICE
+
+if [ "${CHOICE}" != "Y" ]; then
```
Comment (MasterKenth) [inline scripts/put-kms-key-policy.sh:26]: Could we check for lower-case `y` as well? Optional.

**Review by MasterKenth — CHANGES_REQUESTED** [review]

File: scripts/put-kms-key-policy.sh (hunk 3 above)
Comment (kajm) [reply scripts/put-kms-key-policy.sh:26]: Fixed

File: scripts/put-kms-key-policy.sh (hunk 1 above)
Comment (kajm) [reply scripts/put-kms-key-policy.sh:76]: Fixed

File: README.md (hunk 2 above)
Comment (kajm) [reply README.md:208]: fixed

**Review by MasterKenth — APPROVED** [review]

**Review by dannil76 — APPROVED** [review]

**Pull request merged**